src/Microsoft.Tye.Core/ProjectServiceBuilder.cs

```diff
@@ -28,9 +28,24 @@ public ProjectServiceBuilder(string name, FileInfo projectFile)
         // This is always set on the ApplicationFactory codepath.
         public string TargetFramework { get; set; } = default!;
 
+        // This is always set on the ApplicationFactory codepath.
+        public string[] TargetFrameworks { get; set; } = default!;
```

Comment:
Ew, the default! isn't pretty.

Reply:
get used to it. It's either massive constructor or this 😆